tests/test_reader.py

```diff
@@ -14,6 +14,10 @@
 from numpy.typing import NDArray
```

Comment:
How about testing the `bytes_to_int()` and  `number_abbrev_to_int()` with various parameter as part of `test_util.py` instead of initializing Reader class and validating the same ?